src/Directory.Build.props

```diff
@@ -5,6 +5,11 @@
   <!-- This file is imported by all projects at the beginning of the project files -->
   <Import Project="$(RepoToolsetDir)Settings.props" />
 
+  <PropertyGroup>
+    <IncludeSymbols>true</IncludeSymbols>
```

Comment:
Doesn't using embedded PDBs kind of defeat the whole point of creating the symbol package? We want the actual runtime assemblies to be as small as possible but still allow us, CSS and eventually outside devs to get the symbols (via the myget symbol server) the runtime/app/core dump they are debugging.